### PR TITLE
Add template for the 0.12 release

### PR DESCRIPTION
**Pull request by DataTriny**
Follow up of amethyst/amethyst#1806.

# Addition
- A new `0.12.0` template folder

# Modification
- Renamed the `resources` folder of this template to `config`, and renamed the `display_config.ron` inside it,
- Modified the content of `main.rs` to make use of the new `RenderingBundle`.

# Testing performed
- Issued `amethyst new mygame`,
- Ran `cargo run --features "vulkan"` and made sure everything worked as expected.

# Open question
Should this new template create an empty `assets` directory? It seems impossible at the moment.

CC @jojolepro, @torkleyy.

### CODE REVIEW

**Review by AnneKitsune — APPROVED** [review]

**Comment by BradLugo** [comment]
Is there anything blocking this from being merged? Looks like we're getting some other reports about this being an issue (#91)

**Comment by AnneKitsune** [comment]
Was waiting for a second review but no one did, sorry :P

**Pull request merged**
